.github/workflows/pr-build-live-branch.yml

```diff
@@ -113,8 +113,9 @@ jobs:
 
             - name: Comment on PR with WordPress Playground details
               uses: actions/github-script@v7
-              if: steps.prepare.outputs.plugin-data != '{}'
+              if: steps.prepare.outputs.plugin-data != '{}' && ! github.event.pull_request.head.repo.fork
```

Comment:
This one is subject to reverting back to allowing failures instead of the condition changes - some community PRs still have this step passing.